### PR TITLE
Added a bunch of tests for courses endpoints

### PR DESCRIPTION
**Pull request by DonIsaac**

### CODE REVIEW

File: Gemfile
Comment (McIntireEvan) [inline Gemfile:32]: is removing this intentional?

File: tests/v1/courses/courses_spec.rb
Comment (McIntireEvan) [inline tests/v1/courses/courses_spec.rb:20]: either uncomment or delete

File: tests/v0/map_spec.rb
Comment (McIntireEvan) [inline tests/v0/map_spec.rb:4]: good fix - I see for map spec v1, you add a leading `/`, can you do that for this/others to be consistent?

**Review by McIntireEvan — APPROVED** [review]
I left a few comments, but overall looks good. Approving to unblock you, but would like to see those addressed before merging.

Thanks for taking testing so seriously!

File: Gemfile
Comment (DonIsaac) [reply Gemfile:32]: yes

File: tests/v0/map_spec.rb
Comment (DonIsaac) [reply tests/v0/map_spec.rb:4]: yes

**Review by McIntireEvan — APPROVED** [review]

**Pull request merged**
